content/logs/log_collection/_index.md

```diff
@@ -129,40 +107,80 @@ logs:
 [9]: /agent/faq/agent-configuration-files/
 
 {{% /tab %}}
-{{% tab "Stream logs from JournalD" %}}
-
-Set `type` to **journald** then specify the `path` of your journal.
-
-Example:
+{{% tab "Stream logs from journald" %}}
 
 To gather logs from JournalD, create a `journald.d/conf.yaml` file at the root of your [Agent's configuration directory][9] with the following content:
 
-```
+```yaml
 logs:
   - type: journald
     path: /var/log/journal/
     include_units:
```

Comment:
Let's remove the include_units section for this doc. This example is covered in the specific journald integration documentation.